modules/openbsd/openbsd-driver.c

```diff
@@ -0,0 +1,135 @@
+#include <sys/ioctl.h>
+#include <fcntl.h>
+#include <errno.h>
+#include <unistd.h>
+
+#include "openbsd-driver.h"
+
+#include "logsource.h"
+#include "messages.h"
+#include "logreader.h"
+#include "fdhelpers.h"
+#include "apphook.h"
+#include "stats/stats-registry.h"
+#include "poll-fd-events.h"
+#include "logproto/logproto-dgram-server.h"
+#include "transport/transport-socket.h"
+
+#ifndef LIOCSFD
+#define LIOCSFD   _IOW('l', 127, int) /* set sendsyslog() fd */
```

Comment:
This was conditionally defined because the LIOCSFD is defined on openBSD only. When I compiled on Linux it failed. To be able to remove this define I need update build scripts to exclude this driver compile on other systems than openBSD.
Working on that ...

Reply:
Done: I updated the build scripts to build this driver only on OpenBSD